src/listeners/new-suggestion.ts

```diff
@@ -0,0 +1,29 @@
+import { Suggestion } from "../structures/suggestion.ts";
+import type { Listener } from "../types/listener.ts";
+import { getConfig } from "../utils.ts";
+
+export default ({
+	event: "messageCreate",
+	async handler(message) {
+		if (
+			!message.inGuild() ||
+			!message.deletable ||
+			!message.member ||
+			message.author.bot
+		)
+			return;
+
+		const dbConfig = getConfig.get({ guildId: message.guildId });
+
+		if (!dbConfig?.suggestionChannel) return;
```

Comment:
at some point we might want to add some basic filtering properties to events like these to reduce the boilerplate required, but that can be done later